MovieCatalogCrawler/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
+spring.datasource.username=asd
+spring.datasource.password=asd
```

Comment:
Do we need those properties?

Reply:
These properties are for the h2 database and if you want to run tests at some point, we need them.